returnn/tf/util/data.py

```diff
@@ -34,17 +34,17 @@ class Types:
     Defines possible values for ``kind``.
     """
     Unspecified = None
-    Batch = "batch"
-    Spatial = "spatial"  # also time
-    Time = "spatial"  # we don't treat this as different
-    Feature = "feature"
+    Batch = Entity("batch")
+    Spatial = Entity("spatial")  # also time
+    Time = Entity("spatial")  # we don't treat this as different
```

Comment:
This comment is no longer true now, is it?

Reply:
Whoops, good point!

Reply:
Ok fixed. But strangely all the tests also passed without this fix. Maybe `Time` is not often (or not at all) used.